specs/engage-digital_openapi3.yaml

```diff
@@ -5,7 +5,15 @@ info:
   title: RingCentral Engage Digital API
   version: '1.0'
 servers:
-  - url: 'https://<yourdomain>.api.engagement.dimelo.com/1.0'
+  - url: 'https://{account_name}.api.{platform_hostname}/1.0'
+    description: API server endpoint
+    variables:
+      account_name:
+        default: domain-name
+        description: Your own account name used as subdomain, it's the same as in the url of the engage digital service.
+      platform_hostname:
+        default: api.engagement.dimelo.com

```

Comment:
If you agreed on the `.api` hardcoded in the url, then you can remove it from here. Also from the description.